projects/1.20/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2471 @@
+  "tetra.stats.silkTouch.tooltip": "破坏时掉落方块本身，而不是通常的掉落物",
+  "tetra.stats.replanting": "Replanting",
+  "tetra.stats.replanting.tooltip": "横扫收集不会破坏未长成的农作物，而被破坏的农作物则会自动补种",
+  "tetra.stats.fortune": "时运",
+  "tetra.stats.fortune.tooltip": "在采集或回收时增加掉落概率",
+  "tetra.stats.flame": "火矢",
+  "tetra.stats.flame.tooltip": "击中实体时使其着火§e%s§r秒，弹射物会点燃TNT、营火一类的方块",
+  "tetra.stats.punch": "冲击",
+  "tetra.stats.punch.tooltip": "击退距离§e+%s§r倍",
+  "tetra.stats.infinity": "无限",
+  "tetra.stats.infinity.tooltip": "射击时不再消耗普通的箭，但依然会消耗特殊的箭",
+  "tetra.stats.quickStrike": "迅捷",
+  "tetra.stats.quickStrike.tooltip": "在攻击冷却时将最小伤害降低至§e%s§r",
+  "tetra.stats.counterweight": "配重",
+  "tetra.stats.counterweight.tooltip": "如果物品总计使用了§e%s§r点完整度，攻击速度§e+0.15§r。每相差1点增益减少0.05",
+  "tetra.stats.counterweight.indicator_tooltip": "当前增益：§e%s\n \n§7如果物品总计使用了§e%s§r点完整度，攻击速度§e+0.15§r。每相差1点增益减少0.05",
+  "tetra.stats.softStrike": "轻击",
+  "tetra.stats.softStrike.tooltip": "使用此工具制作的物品获得额外的耐久度",
+  "tetra.stats.fierySelf": "燃烧",
+  "tetra.stats.fierySelf.tooltip_short": "使用时持有者有概率着火，在寒冷群系中概率较低",
+  "tetra.stats.fierySelf.tooltip": "使用时持有者有概率着火，在寒冷群系中概率较低\n\n§7基础概率：§e%s§7\n着火持续时间：§e%s",
+  "tetra.stats.enderReverb": "末影回响",
+  "tetra.stats.enderReverb.tooltip_short": "使用时激怒附近的末影人，并且附近的末影人传送时持有者有概率被一起传送",
+  "tetra.stats.enderReverb.tooltip": "使用时激怒附近的末影人，并且附近的末影人传送时持有者有概率被一起传送\n\n§7基础概率：§e%s§7",
+  "tetra.stats.sculkTaint": "幽匿侵染",
+  "tetra.stats.sculkTaint.tooltip": "破坏方块或击中实体时有§e%s§r的概率产生幽匿脉络",
+  "tetra.stats.sculkTaint.tooltip_extended": "与在幽匿催发体旁击杀实体的效果相似",
+  "tetra.stats.toolbelt.quickslot": "快捷栏",
+  "tetra.stats.toolbelt.quickslot.tooltip": "有§e%s§r个快捷栏槽位",
+  "tetra.stats.toolbelt.potion_storage": "药水栏",
+  "tetra.stats.toolbelt.potion_storage.tooltip": "有§e%s§r个药水槽位",
+  "tetra.stats.toolbelt.storage": "储物栏",
+  "tetra.stats.toolbelt.storage.tooltip": "有§e%s§r个储物槽位",
+  "tetra.stats.toolbelt.quiver": "箭袋栏",
+  "tetra.stats.toolbelt.quiver.tooltip": "有§e%s§r个箭袋槽位",
+  "tetra.stats.toolbelt.booster": "力量提升",
+  "tetra.stats.toolbelt.booster.tooltip": "提供强度为§e%s§r的跳跃提升和有限制的飞行\n \n§7[跳跃] (半空)§8积累跳跃提升\n§7[潜行] + [跳跃]§8跳起",
+  "tetra.stats.toolbelt.suspend_self": "悬浮",
+  "tetra.stats.toolbelt.suspend_self.tooltip": "半空中跳跃使穿戴者悬停，只在穿戴者受信标或潮涌核心影响产生作用\n \n§7[跳跃] (半空)§8悬停\n§7[潜行]§8缓慢下降\n§7[潜行] + [跳跃]§8取消悬停",
+  "tetra.stats.toolbelt.suspend_self.tooltip_extended": "注意：潮涌核心只影响潮湿的实体，因此受其影响的穿戴者只能在下雨时悬停",
+  "tetra.stats.bow.releaseLatch": "释放销",
+  "tetra.stats.bow.releaseLatch.tooltip": "在弓被拉满时自动射箭",
+  "tetra.stats.bow.overbowed": "重弓",
+  "tetra.stats.bow.overbowed.tooltip": "这把弓拉起来很费力，弓只能被持续拉满§7§e%s§r秒",
+  "tetra.stats.bow.overbowed.tooltip_extended": "超出这个上限时弓会被松开并且不射出弓箭。弓会进入§70.5§r秒的冷却中",
+  "tetra.stats.multishot": "多重射击",
+  "tetra.stats.multishot.level": "多重射击数量",
+  "tetra.stats.multishot.efficiency": "多重射击散布",
+  "tetra.stats.multishot.tooltip": "每发射出多个弹射物，沿水平方向散布\n \n§7数量：§e%s\n§7散布：§e%s",
+  "tetra.stats.zoom": "放大",
+  "tetra.stats.zoom.tooltip": "当瞄准或按住[Shift]时将画面放大§e%sx§r倍",

```

| **审查级别**    |
|:-:|
| ❓疑问 / 需要作者确认 |

**原文: Zooms in by §e%sx§r while holding crouching**

```suggestion
  "tetra.stats.zoom.tooltip": "当瞄准或潜行时将画面放大§e%sx§r倍",
```

---

后续的一些翻译用的是潜行，感觉可以统一一下

---

```suggestion
  "tetra.stats.zoom.tooltip": "瞄准时潜行会将画面放大§e%sx§r倍",
```
感觉我一开始就翻错了